sass/themes/cr/2019/components/full-height-single-image-single-copy/full-height-single-image-single-copy.twig

```diff
@@ -0,0 +1,14 @@
+<section class="paragraph--full-height-single-image-single-copy {{modifier_class}} bg--coral">
+  <div class="fhsisc__image-wrapper objectfit">
+     <img src="kss-assets/images/spice_3-4_2x.jpg">
```

Comment:
Was gonna suggest just using the picture element to load a 4:3 image for mobile if you've got time